tests/title-show-test.js

```diff
@@ -43,6 +48,18 @@ describeApplication('TitleShow', function() {
       expect(TitleShowPage.publicationType).to.equal(title.pubType);
     });
 
+    it('displays the authors', function() {
+      expect(TitleShowPage.contributorsList[0]).to.equal('AuthorsSally Writer, Jane Wordsmith');
```

Comment:
there's a bunch of places where we do this sort of thing, mangling an element to weird text and then testing against it.  works for now, but we might want some kind of better matchers down the line